src/sentry/static/sentry/app/views/settings/components/dataPrivacyRulesPanel/dataPrivacyRulesPanelForm/dataPrivacyRulesPanelFormField.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+
+import {IconQuestion} from 'app/icons/iconQuestion';
+import Tooltip from 'app/components/tooltip';
+import space from 'app/styles/space';
+
+type Props = {
+  label: string;
+  tooltipInfo: string;
+  children: React.ReactNode;
+  isFullWidth?: boolean;
+};
+
+const DataPrivacyRulesPanelFormField = ({
+  label,
+  tooltipInfo,
+  children,
+  isFullWidth,
+}: Props) => (
+  <Wrapper isFullWidth={isFullWidth}>
+    <Label>
+      <LabelDescription>{label}</LabelDescription>
+      <Tooltip title={tooltipInfo} position="top">
+        <IconQuestion color="gray1" />
```

Comment:
Should this be a theme reference?

Reply:
I have updated the IconQuestion ... now it's `const color = theme[providedColor] ?? providedColor;`